src/praisonai/praisonai/cli/legacy/interactive_legacy.py

```
@@ -0,0 +1,1492 @@
+                        warnings.filterwarnings("ignore")
+                        
+                        model = session_state.get('current_model')
+                        conversation_history = session_state.get('conversation_history', [])
+                        
+                        live_status.update_status("Creating agent...")
+                        
+                        # Build backstory with context
+                        backstory = "You are a helpful AI assistant with access to tools for file operations, code intelligence, and shell commands."
+
+                        # Auto-load AGENTS.md/CLAUDE.md project context (unless --no-context)
+                        if not getattr(args, 'no_context', False):
```

Ruff (0.15.20)

[error] Undefined name `args`

(F821)